### PR TITLE
Use JsonPath $ as the outer level object

### PR DESCRIPTION
**Pull request by filipecosta90**

### CODE REVIEW

File: tests/benchmarks/json_set_ResultSet.totalResultsAvailable_1_jsonsl-yahoo2_json.yml [hunk 1]
```diff
@@ -14,4 +14,4 @@ clientconfig:
     - requests: 5000000
     - threads: 2
     - pipeline: 1
-    - command: 'JSON.GET jsonsl-yahoo2 ResultSet.totalResultsAvailable 1'
+    - command: 'JSON.SET jsonsl-yahoo2 $.ResultSet.totalResultsAvailable 1'
```
Comment (gkorland) [inline tests/benchmarks/json_set_ResultSet.totalResultsAvailable_1_jsonsl-yahoo2_json.yml:17]: @filipecosta90 it seems like a mistake should it be JSON.GET or SET?

File: tests/benchmarks/json_set_message.code_1_jsonsl-yelp_json.yml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 version: 0.2
 name: "json_set_message.code_1_jsonsl-yelp_json"
-description: "JSON.GET jsonsl-yelp message.code || https://oss.redislabs.com/redisjson/performance/"
+description: "JSON.GET jsonsl-yelp $.message.code || https://oss.redislabs.com/redisjson/performance/"
```
Comment (gkorland) [inline tests/benchmarks/json_set_message.code_1_jsonsl-yelp_json.yml:3]: @filipecosta90 the description doesn't match the command

File: tests/benchmarks/json_vs_hashes_hset_key_simple.yml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 version: 0.2
 name: "json_vs_hashes_hset_key_simple"
-description: 'HSET key_hash field1 value1 field2 value2 || Use-case to compare against JSON.SET key_json . {"field1":"value1","field2":"value2"}'
+description: 'HSET key_hash field1 value1 field2 value2 || Use-case to compare against JSON.SET key_json $. {"field1":"value1","field2":"value2"}'
```
Comment (gkorland) [inline tests/benchmarks/json_vs_hashes_hset_key_simple.yml:3]: @filipecosta90 the command is still in the old format

**Review by gkorland — CHANGES_REQUESTED** [review]

File: tests/benchmarks/json_set_ResultSet.totalResultsAvailable_1_jsonsl-yahoo2_json.yml (hunk 1 above)
Comment (filipecosta90) [reply tests/benchmarks/json_set_ResultSet.totalResultsAvailable_1_jsonsl-yahoo2_json.yml:17]: it should be json.set. This is a fix

Comment (gkorland) [reply tests/benchmarks/json_set_ResultSet.totalResultsAvailable_1_jsonsl-yahoo2_json.yml:17]: @filipecosta90 but the description still says `JSON.GET jsonsl-yahoo2`

File: tests/benchmarks/json_vs_hashes_hset_key_simple.yml (hunk 3 above)
Comment (gkorland) [reply tests/benchmarks/json_vs_hashes_hset_key_simple.yml]: @filipecosta90 it still seems like the description is a bit strange 
```
Use-case to compare against JSON.SET key_json $. {"field1":"value1","field2":"value2"}'
```

**Review by gkorland — APPROVED** [review]

**Pull request merged**